composer.json

```diff
@@ -32,13 +32,13 @@
         "sonata-project/admin-bundle": "^3.88",
         "sonata-project/exporter": "^1.11.0 || ^2.0",
         "sonata-project/form-extensions": "^0.1 || ^1.4",
-        "symfony/config": "^4.4",
+        "symfony/config": "^4.4 || ^5.2",
         "symfony/console": "^4.4",
-        "symfony/dependency-injection": "^4.4",
+        "symfony/dependency-injection": "^4.4 || ^5.2",
         "symfony/doctrine-bridge": "^4.4",
         "symfony/form": "^4.4",
         "symfony/framework-bundle": "^4.4",
-        "symfony/http-foundation": "^4.4",
+        "symfony/http-foundation": "^4.4 || ^5.2",
         "symfony/http-kernel": "^4.4",
         "symfony/options-resolver": "^4.4 || ^5.1",
```

Comment:
Should you update the 5.1 ?

Reply:
If 5.1 works, why?

Reply:
So why using `^5.2` for others ?

We could even use `^5.0` every time.
I thought our policy was to allow only supported version. https://symfony.com/releases

Reply:
Sure but this Pr is for supporting new versions, why should we raise the bar for a version which we already support?

Reply:
If you agree, I can use 5.1 or 5.2 based on the decision we take.
Please, let me know what your suggestions are in order to get this PR merged.

Reply:
I don't want to be too picky, I am fine with a bump to 5.2

Reply:
I dont really have preference about the one chosen but Id like to have consistency :)

Reply:
Thank you!